packages/compiler/lib/decorators.ts

```diff
@@ -821,15 +821,6 @@ export function getKeyName(program: Program, property: ModelProperty): string {
   return program.stateMap(keyKey).get(property);
 }
 
-/**
```

Comment:
is this removed for testing?

Reply:
No it was removed because we said we should not be duplicating the documentation on both the `$method` form and the `extern dec method` and should just put the docstring on the `extern dec method`.

Reply:
ah interesting, I didn't know we decided that.